Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM caddy:2-alpine
+COPY dist/ /var/www/html
+COPY Caddyfile /etc/caddy/Caddyfile
```

Comment:
It would be worth it to add important files for the Caddy server in the README for easier documentation